README.md

```diff
@@ -1,5 +1,121 @@
-# Balance Tetris
+<div style="background-color: #FFFFFF">
+    <p align="center">
+    <a href="https://github.com/WoochanLee/BalanceTetris">
+        <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/young.png?raw=true" height="300" />
+        <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/chan.png?raw=true" height="300" />
+    </a>
+    </p>
+    <br>
+    <p align="center">
+        <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/logo.png?raw=true" width="600"/>
+    </p>
+    <p align="center" style="color: #000; font-size: 25px">🕹️You Can Single Play Online👇</p>
+    <div align="center" style="margin-bottom: 40px">
+        <a href="https://github.com/WoochanLee/BalanceTetris" style="color: blue; font-size: 25px">
+            <u>https://github.com/WoochanLee/BalanceTetris</u>
+        </a>
+    </div>
+    <p align="center">
+        <img src="https://img.shields.io/badge/license-GPL%20v3-black.svg">
+        <img src="https://img.shields.io/badge/release-alpha-black.svg">
+    <br />
+</div>
+<br>
+<br>
 
-Made by Woody
+# Features 🎮
 
-Play Tetris Link : [https://woochanlee.github.io/BalanceTetris/](https://woochanlee.github.io/BalanceTetris/)
+Easy to Play: You can play Tetris anywhere with a computer web browser.
+<br>
+Using T-spin: You can use T-spin technic.
+<br>
+Cats are cute: Cats are cute.
+<br>
+<br>
+<br>
+
+# How to Play 💁
+
+<b style="font-size: 15px">Q & E</b> or <b style="font-size: 15px">Z & C</b> --> Rotate the Control Block ( or you can use ↑ key )
+<br>
+<br>
+<b style="font-size: 20px">← ↓ →</b> --> Move the Control Block
+<br>
+<br>
+<b style="font-size: 15px">Space Bar</b> --> Lower the Control Block to the floor at once
+<br>
+<br>
+<b style="font-size: 15px">Left Shift</b> --> Save or eject blocks to temporary storage
+<br>
+<br>
+<br>
+
+# Contributing 🙋
```

Comment:
This part seems "General" explanation. 
I think this need more detail guidelines to contribute this repository. (I.e. Required 2 LGTM)

Reply:
How about this way.

## Contributing
You could participate to improve and move forward Balance Tetris.

### Code of conduct
 (Here's part is up to you)
### Contribution Guideline
(Fork, Push, Pull-Request steps to `develop` branch.) 
~~ Explains ~~ 
To merge your commit, a single pull-request strictly required 2 LGTMs (Approves)
If you have permission to merge approved pull-request, You should merge the PR which is approved.
### Issues Guidelines
For bug:
```
What steps will reproduce the problem?
(1)
(2)
(3)

What is the expected result?


What happens instead?
```
For Feature:

```
Something else 
```